### PR TITLE
Some small fixes

### LINKED ISSUE #65: with-lock not working with Python 2
`with-lock` uses `subprocess.run`, which is not present in Python 2

### PR DESCRIPTION
**Pull request by viktordick**
Fixes #65  and #67. Shows docstring of each subcommand when calling with `--help`

### CODE REVIEW

File: perfact/zodbsync/commands/with_lock.py [hunk 1]
```diff
@@ -20,4 +18,4 @@ def add_args(parser):
 
     @SubCommand.with_lock
     def run(self):
-        subprocess.run(self.args.command, shell=True)
+        subprocess.check_call(self.args.command, shell=True)
```
Comment (jan-jockusch) [inline perfact/zodbsync/commands/with_lock.py:21]: This changes the behavior, does it not? Can this be a problem, e.g., if commands regularly return non-zero exit codes?

**Review by jan-jockusch — APPROVED** [review]
One question regarding `subprocess.check_call`, otherwise LGTM

File: perfact/zodbsync/commands/with_lock.py (hunk 1 above)
Comment (viktordick) [reply perfact/zodbsync/commands/with_lock.py:21]: Well the only thing that happens after this call is the release of the lock, which also happens if the program crashes.

AFAIK no one uses this feature yet. I tried to use it once and found that it does not work with Python 2. In general I think raising an error is better that ignoring it. If the return code should be ignored, one can always use `|| true`.

Comment (jan-jockusch) [reply perfact/zodbsync/commands/with_lock.py:21]: I thought as much, but I was guessing… thanks for clearing that up!

My review stands, then.

**Pull request merged**